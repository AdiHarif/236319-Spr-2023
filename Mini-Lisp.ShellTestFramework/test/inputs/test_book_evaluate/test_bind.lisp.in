(bind '(a b) '(x y) '())
(bind '(a b) '(x y) '())
(bind '(a b) '(x y) '(c.T z.T))
(bind)
(bind '(a b) '(x y z) '())
(bind '(a b c) '(x y) '())
