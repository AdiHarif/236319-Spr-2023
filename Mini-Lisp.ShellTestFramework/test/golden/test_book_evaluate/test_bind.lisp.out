((A.X) (B.Y))
((A.X) (B.Y))
((A.X) (B.Y) (C.T) (Z.T))
Error [MISSING] in (FORMALS ACTUALS A-LIST)
Error [INVALID] in (MISSING-FORMALS)
Error [INVALID] in (MISSING-ACTUALS)
